apps/styleguide/app/nav-bar/nav-bar.tsx

```diff
@@ -5,7 +5,7 @@ import { NavBarItem } from './nav-bar-item'
 export function NavBar() {
   return (
     <div className={styles['nav-bar']}>
-      <h1 className={styles['nav-bar-title']}>React Bedrock</h1>
+      <h3 className={styles['nav-bar-title']}>React Bedrock</h3>
```

Comment:
small note here--people don't really follow this anymore, but I've seen older designers get frustrated when people skip h1 for a more nested header. People generally do it to get the styling they want, but it kinda breaks the html structure

Reply:
Yeah... I think it might not be a great thing to do long term for accessibility reasons with screen readers. Right now the Title would be massive with the styling that I used for the h1, but I think later I will try to update the styling to be smaller for h1 and convert to using that instead. Or maybe just override the h1 in my elements override